tools/win/generate_breakpad_symbols.py

```diff
@@ -4,38 +4,66 @@
 # Copyright (c) 2013 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
-
 """Convert pdb to sym for given directories"""
 
 import errno
 import glob
 import argparse
 import os
-import queue
 import re
-import subprocess
 import sys
-import threading
+import asyncio
+import shutil
 
 from datetime import datetime
 from shutil import rmtree
 
-CONCURRENT_TASKS=1
-BRAVE_ROOT=os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
-
-
-def GetCommandOutput(command):
-    """Runs the command list, returning its output.
-
-    Prints the given command (which should be a list of one or more strings),
-    then runs it and returns its output (stdout) as a string.
-
-    From chromium_utils.
-    """
-    devnull = open(os.devnull, 'w') # pylint: disable=consider-using-with
-    proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=devnull) # pylint: disable=consider-using-with
-    output = proc.communicate()[0]
-    return output.decode('utf-8')
+BRAVE_ROOT = os.path.abspath(
+    os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+
+
+async def ProcessBinary(semaphore, options, binary):
+    dump_syms = os.path.join(options.build_dir, 'dump_syms.exe')
+    sym_temp_output = binary + '.sym'
+    async with semaphore:
+        start_time = datetime.utcnow()
+        if options.verbose:
+            print("Generating symbols for {0}".format(binary))
+        with open(sym_temp_output, 'w') as output:
+            process = await asyncio.create_subprocess_exec(
+                dump_syms,
+                binary,
+                stdout=output,
+                stderr=asyncio.subprocess.PIPE)
+            _, stderr = await process.communicate()
+
+            # TODO(atuchin): investigate why dump_syms.exe fails
+            # and replace to False
+            # if process.returncode != 0:
+            #     decoded_stderr = stderr.decode('utf-8')
+            #     return False, \
+            #            f"dump_syms failed for {binary} \n {decoded_stderr}"
```

Comment:
In fact we have a problem here that the old script suppresses: 
` Couldn't locate EXE or DLL file.
Couldn't locate EXE or DLL file.
WriteSymbols failed.`

I suggest filing another issue to investigate it.

Reply:
what about showing errors, but not fail the build? like `Failed, but ignoring the error: ...`.

Reply:
Done